upstream-community-operators/oneagent/dynatrace-monitoring.v0.2.0.clusterseviceversion.yaml

```diff
@@ -0,0 +1,238 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [{
+       "apiVersion": "dynatrace.com/v1alpha1",
+       "kind": "OneAgent",
+       "metadata": {
+          "name": "oneagent",
+          "namespace": "dynatrace"
+       },
+       "spec": {
+          "apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api",
+          "skipCertCheck": false,
+          "tokens": "",
+          "nodeSelector": {},
+          "tolerations": [
+             {
+                "effect": "NoSchedule",
+                "key": "node-role.kubernetes.io/master",
+                "operator": "Exists"
+             }
+          ],
+          "image": "",
+          "args": [
+             "APP_LOG_CONTENT_ACCESS=1"
+          ],
+          "env": []
+       }
+      }]
+    categories: "Monitoring,OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/dynatrace/dynatrace-oneagent-operator:v0.2.0
+    createdAt: 2019-02-06T12:59:59Z
+    description: Install full-stack monitoring of OpenShift clusters with the Dynatrace OneAgent.
```

Comment:
Change the description for upstream, probably shouldn't mention OpenShift

Reply:
File name is not correct. Change to dynatrace-monitoring.v0.2.0.clusterse**r**viceversion.yaml